trio/_core/_generated_run.py

```diff
@@ -193,21 +193,21 @@ def current_trio_token():
                  lock = trio.Lock()
                  await lock.acquire()
                  async with trio.open_nursery() as nursery:
-                     child = nursery.start_soon(lock_taker, lock)
+                     nursery.start_soon(lock_taker, lock)
                      # child hasn't run yet, we have the lock
                      assert lock.locked()
-                     assert lock._owner is trio.current_task()
+                     assert lock._owner is trio.hazmat.current_task()
                      await trio.testing.wait_all_tasks_blocked()
                      # now the child has run and is blocked on lock.acquire(), we
                      # still have the lock
                      assert lock.locked()
-                     assert lock._owner is trio.current_task()
+                     assert lock._owner is trio.hazmat.current_task()
                      lock.release()
                      try:
                          # The child has a prior claim, so we can't have it
                          lock.acquire_nowait()
                      except trio.WouldBlock:
-                         assert lock._owner is child
+                         assert lock._owner is not trio.hazmat.current_task()
```

Comment:
Unfortunately, this file is automatically generated code... the real docstring is in `trio/_core/_run.py`. So the process to fix it is:

1. make the edits to `trio/_core/_run.py`
2. run `python trio/_tools/gen_exports.py` to regenerate this file
3. commit the changes to both `trio/_core/_run.py` and `trio/_core/_generated_run.py`

Reply:
Oops, I missed that information 😆 OK I changed it